okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/UrlSelectorImpl.java

```diff
@@ -176,30 +177,40 @@ private static String switchWsToHttp(String url) {
         return redirectTo(current, httpUrls.get((currentUrl.get() + 1) % httpUrls.size()));
     }
 
+    @Override
+    public void markAsSucceeded(HttpUrl failedUrl) {
+        if (useFailedUrlCache) {
```

Comment:
Can we just remove this? Updating this cache should be pretty inexpensive

Reply:
seems fine, it's just disabling it in cases where it doesn't make sense.
Although it's a bit weird we accept a negative value here...